packages/fetch-retry/test.js

```diff
@@ -1,40 +1,41 @@
-const assert = require('assert');
-const {createServer} = require('http');
-const AbortController = require('abort-controller');
-const setup = require('./index');
+/* eslint-env jest*/
```

Comment:
We export a config for Jest in our style guide, which you can set up using overrides :)

The style guide is heavily optimised for TS though, so let me know if you hit any issues.